xbmc/guilib/GUIWindow.cpp

```diff
@@ -123,7 +121,7 @@ bool CGUIWindow::Load(const std::string& strFileName, bool bContainsPath)
 bool CGUIWindow::LoadXML(const std::string &strPath, const std::string &strLowerPath)
 {
   // load window xml if we don't have it stored yet
-  if (!m_windowXMLRootElement)
+  if (m_windowXMLRootElement == nullptr)
```

Comment:
Why not stick with how it was - `if (!m_win...` ? `std::unique_ptr` has `bool` operator as far as i know.